src/assets/scripts/client/constants/inputConstants.js

```diff
@@ -72,5 +72,6 @@ export const PARSED_COMMAND_NAME = {
     CLEAR: 'clear',
     AIRPORT: 'airport',
     RATE: 'rate',
-    TRANSMIT: 'transmit'
+    TRANSMIT: 'transmit',
+    AIRAC: 'airac'
```

Comment:
Could you sort these alphabetically? I know it's not from this PR, but I'm not sure why they're in a big heap like this

Reply:
done